fixtures/american-dj/mega-par-profile-plus.json

```diff
@@ -0,0 +1,125 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Mega Par Profile Plus",
+  "categories": ["Color Changer", "Dimmer", "Strobe"],
+  "meta": {
+    "authors": ["Anonymous"],
+    "createDate": "2024-01-07",
+    "lastModifyDate": "2024-01-07"
+  },
+  "links": {
+    "manual": [
+      "https://www.fullcompass.com/common/files/23670-ADJMegaParProfilePlusManual.pdf"
+    ],
+    "productPage": [
+      "https://www.adj.com/mega-par-profile-plus"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=g0ON1EzrZzQ"
+    ]
+  },
+  "rdm": {
+    "modelId": 0
+  },
+  "physical": {
+    "dimensions": [9, 8.75, 3.5],
+    "weight": 1.1,
+    "power": 16,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED"
+    }
+  },
+  "availableChannels": {
+    "Red": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "UV": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "UV"
+      }
+    },
+    "Dimmer": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Strobe": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 31],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Closed"
+        },
+        {
+          "dmxRange": [32, 63],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        },
+        {
+          "dmxRange": [64, 95],
+          "type": "StrobeSpeed",
+          "speed": "slow"
+        },
+        {
+          "dmxRange": [96, 127],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        },
+        {
+          "dmxRange": [128, 255],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        }
+      ]
+    }
+  },
+  "modes": [
+    {
```

Comment:
Also 9/10-chan modes.

Fortunately, this looks almost identical to the "american-dj/mega-hex-par", except without the White and Amber channels.  We can copy channels 8/9/10/11/12 directly from that file (after #3715), including the switched channels.

The only update we'll need to do is redoing the "Color Macros" channel, since those are all different -- there's a table of the 65 colors (!) and their dmx/r/g/b/uv values on p.24 of the manual.  (How do you put r/g/b/uv values into hex?)